lbz/authz.py

```diff
@@ -101,6 +101,15 @@ def set_policy(self, token: str):
         self.allowed_resource = None
         self.denied_resource = None
 
+    def reset_policy(self):
```

Comment:
There is method `set_initial_state` please remove it and insert this (they have same purpose)

Reply:
I'm not sure about this, I think `set_initial_state` is supposed to prepare the authorizer for permissions check (so it also sets `self.action`, while this `reset_policy` is supposed to remove all information about the policy because the request didn't have any authorization data.
Anyway I want to redesign this class, I think the way the authorization is executed can be simplified a lot and afterwards these methods won't be necessary anymore.